Add support for couchbase

prajwal-pai77 opened the pull request.

shyam-cb commented on docs/docs/marketplace/plugins/couchbase.md:
```diff
@@ -0,0 +1,239 @@
+title: Couchbase
+---
+
+ToolJet integrates with Couchbase to utilize its NoSQL database capabilities. This integration enables ToolJet to perform document operations such as creating, reading, updating, and deleting documents, as well as executing SQL++ queries and Full-Text Search (FTS) operations in Couchbase databases.
```
Try to highlight vector search

prajwal-pai77 replied:
Updated readme.

shyam-cb commented on docs/docs/marketplace/plugins/couchbase.md:
```diff
@@ -0,0 +1,239 @@
+- **Collection**: The collection name
+- **Document ID**: The unique identifier of the document to update
+- **Document**: The updated document data as a JSON object
+
```
Add a not as it replaces the document

prajwal-pai77 replied:
Updated readme.

shyam-cb commented on docs/docs/marketplace/plugins/couchbase.md:
```diff
@@ -0,0 +1,239 @@
+- **Bucket**: The name of the bucket to search in
+- **Scope**: The scope name
+- **Index Name**: The name of the FTS index to search against
+- **Search Query**: The FTS search query as a JSON object
```
support for search options

prajwal-pai77 replied:
Currently all the search options are supported via the json request, as discussed

shyam-cb commented on marketplace/plugins/couchbase/lib/index.ts:
```diff
@@ -0,0 +1,101 @@
+    }
+  }
+
+  async getConnection(sourceOptions: SourceOptions): Promise<SourceOptions> {
```
do we need a sync function

prajwal-pai77 replied:
Yes all the other providers have implemented asynchronously

shyam-cb commented on marketplace/plugins/couchbase/lib/query_operations.ts:
```diff
@@ -0,0 +1,172 @@
+  }
+
+  const queryBody = {
+    statement: query,
```
check wether key vaue params suported for the statement

shyam-cb replied:
@prajwal-pai77 any update on this ?

prajwal-pai77 replied:
Done, updated query endpoint as discussed.

RudraPratik30 committed:
wip

slancerk committed:
Added GCP and EC2_AMI

slancerk committed:
Updated docs for ami and docker

slancerk committed:
small fix

RudraPratik30 committed:
privacy policy

RudraPratik30 committed:
debug components

RudraPratik30 committed:
buy ai credits

adishM98 committed:
Merge pull request #14115 from ToolJet/terraform-vm-added

Added GCP and EC2_AMI

RudraPratik30 committed:
gen code wip

RudraPratik30 committed:
Generate Code

RudraPratik30 committed:
update docs version

adishM98 committed:
Update WEBSITE_SIGNUP_URL in workflow configuration

adishM98 committed:
Merge pull request #14133 from ToolJet/adishM98-patch-1

Update WEBSITE_SIGNUP_URL in workflow configuration

RudraPratik30 committed:
remove modify specs

adishM98 committed:
Merge pull request #14131 from ToolJet/docs/ai-sept-release

[docs]: AI Sept Release

RudraPratik30 committed:
wip

RudraPratik30 committed:
[docs]: LDAP Multi OU

adishM98 committed:
Merge pull request #14157 from ToolJet/docs/ldap-ou-sso

[docs]: LDAP Multi OU

RudraPratik30 committed:
[docs]: AI Server URL Whitelisting

adishM98 committed:
Merge pull request #14178 from ToolJet/docs/ai-whitelist

[docs]: AI Server URL Whitelisting

RudraPratik30 committed:
remove slash

adishM98 committed:
Merge pull request #14179 from ToolJet/docs/remove-slash

remove slash

adishM98 committed:
Use npx for Netlify CLI deployment (#14191)

* Use npx for Netlify CLI deployment

Replaced global installation of Netlify CLI with local installation using npx.

* Update Netlify CLI installation method in workflow

adishM98 committed:
reverting netlify changes (#14193)

adishM98 committed:
Add ALLOWED_USER13_TEST_SYSTEM to workflow (#14197)

RudraPratik30 committed:
wip

RudraPratik30 committed:
[docs]: API User Metadata (#14217)

adishM98 committed:
Add ALLOWED_USER14_TEST_SYSTEM to test system

adishM98 committed:
Add ALLOWED_USER14_TEST_SYSTEM to test system (#14225)

adishM98 committed:
Modify branch reference and remove cypress checks

Updated the GitHub Actions workflow to change the branch reference from 'main' to 'lts-3.16' for multiple jobs and removed the cypress vulnerability check section.

adishM98 committed:
Merge pull request #14269 from ToolJet/adishM98-patch-1

Modify branch reference and remove cypress checks

RudraPratik30 committed:
revamp done

RudraPratik30 committed:
build fix

adishM98 committed:
Update workflow scheduling instructions across deployment documentation

adishM98 committed:
Revert "Update workflow scheduling instructions across deployment documentation"

This reverts commit 618ac7399a56c1da2f0c7faf2666e9c0668b58a4.

RudraPratik30 committed:
[docs]: Add Custom Style (#14324)

RudraPratik30 committed:
[docs]: Global App Version (#14328)

RudraPratik30 committed:
[docs]: Preview Access Control for End Users

slancerk committed:
updated render-preview-deploy file

adishM98 committed:
Merge pull request #14352 from ToolJet/updated-render-preview

updated render-preview-deploy file

slancerk committed:
added event.issue.number

adishM98 committed:
Merge pull request #14356 from ToolJet/updated-render-preview

adishM98 committed:
Merge pull request #14350 from ToolJet/docs/end-user-app-preview

[docs]: Preview Access Control for End Users

RudraPratik30 committed:
[docs]: App Builder Feedback Update (#14327)

* nav bar changes

* update custom themes

* wip

* feedback complete

adishM98 committed:
Disable scheduled cron job for LTS updates

Comment out the scheduled cron job for LTS updates.

adishM98 committed:
Merge pull request #14393 from ToolJet/adishM98-patch-1

Disable scheduled cron job for LTS updates

RudraPratik30 committed:
[docs]: Update Gemini Model

adishM98 committed:
Merge pull request #14402 from ToolJet/docs/gemini-model-update

[docs]: Update Gemini Model

RudraPratik30 committed:
wip waiting for rest api changes

adishM98 committed:
Merge pull request #14416 from ToolJet/docs/marketplace-sprint-13

[docs]: Marketplace Sprint 13

RudraPratik30 committed:
[docs]: ToolJet API cURL (#14338)

* [docs]: ToolJet API cURL

* testing updates

slancerk committed:
Use checkout-branch default

adishM98 committed:
Merge pull request #14436 from ToolJet/render-preview-fix

Render-preview-deploy file FIX

slancerk committed:
modified to target

adishM98 committed:
modified to target (#14445)

adishM98 committed:
Add Cloudflare cache purge step with user authorization check to develop branch

adishM98 committed:
Merge pull request #14446 from ToolJet/fix/api-cache-develop

Add Cloudflare cache purge step with user authorization check to develop branch

slancerk committed:
added fix

slancerk committed:
Merge branch 'develop' into render-preview-fix

slancerk committed:
Update render-preview-deploy.yml

slancerk committed:
updated fix

slancerk committed:
small fix

adishM98 committed:
Merge pull request #14448 from ToolJet/render-preview-fix

Render preview fix

slancerk committed:
small fix

RudraPratik30 committed:
[docs]: File Picker Component

adishM98 committed:
Merge pull request #14459 from ToolJet/docs/filepicker-component

[docs]: File Picker Component

adishM98 committed:
Merge pull request #14458 from ToolJet/azure-container-entrypoint-fix

[Doc] Azure container apps V3.16 LTS

RudraPratik30 committed:
[docs]: Minor Changes

adishM98 committed:
Merge pull request #14460 from ToolJet/docs/minor-changes-oct-2025

[docs]: Minor Changes

RudraPratik30 committed:
[docs]: Update Support Mail

RudraPratik30 committed:
[docs]: Marketing Script

adishM98 committed:
Merge pull request #14474 from ToolJet/docs/marketing-tracking-script

[docs]: Marketing Script

adishM98 committed:
Merge pull request #14462 from ToolJet/docs/update-support-email

[docs]: Update Support Mail

RudraPratik30 committed:
[docs]: Prompting 101 (#14322)

* [docs]: Prompting 101

* feedback update

RudraPratik30 committed:
[docs]: OIDC PKCE (#14476)

adishM98 committed:
[docs]: Streaming Rsyslog Audit Logs to Datadog (#14384)

* Add guide for streaming Rsyslog audit logs to Datadog

* add use case, add doc to sidebar

* feedback update

---------

Co-authored-by: rudrapratik30 <pratik104agrawal@gmail.com>

adishM98 committed:
docs: Add workflow scheduling instructions and Redis configuration guidance across deployment documentation (#14307)

* Update workflow scheduling instructions across multiple deployment documents

* refactor docs for support new workflow style with redis

* Add warning about whitelisting ToolJet AI features in DigitalOcean and Try ToolJet setup documentation

* Remove "High-Level" from the "How It Works" section title in the migration guide

* Add note about ToolJet version compatibility in migration guide

* Add migration guide reference for users transitioning from Temporal-based workflows

* Update documentation to recommend external Redis for multiple workflow workers across various deployment setups

* Update Docker setup instructions and clarify migration guide version compatibility

* Remove rollback plan section from the Temporal to BullMQ migration guide

* Update documentation to clarify external Redis requirements for multiple worker setups

* Remove TOOLJET_QUEUE_DASH_PASSWORD references from environment variable documentation across multiple setup guides

* Update Redis configuration documentation to clarify external instance requirements and add workflow scheduling variables

* Update migration guide to reflect correct ToolJet version for BullMQ transition

karan-rathod-316 committed:
add domain change details

karan-rathod-316 committed:
disable current version

adishM98 committed:
Merge pull request #14588 from ToolJet/docs/domain-change

[docs] Add domain change details

adishM98 committed:
Update manual Docker build workflow descriptions

adishM98 committed:
Merge pull request #14597 from ToolJet/adishM98-patch-1

Update manual Docker build workflow descriptions

adishM98 committed:
docs: Add observability documentation (#14419)

* docs: Add observability documentation

* minor formatting update

* Enhance OpenTelemetry configuration details and warnings for high cardinality metrics

---------

Co-authored-by: rudrapratik30 <pratik104agrawal@gmail.com>

adishM98 committed:
Modify vulnerability CI schedule and notifications

Updated the vulnerability CI workflow to run weekly instead of bi-weekly. Enhanced Slack notifications with structured payloads and added output retention for audit reports.

adishM98 committed:
Merge pull request #14600 from ToolJet/adishM98-patch-1

Modify vulnerability CI schedule and notifications

adishM98 committed:
Refactor workflow input descriptions and cleanup steps

Updated descriptions in the workflow inputs and added disk cleanup steps.

adishM98 committed:
Merge pull request #14602 from ToolJet/adishM98-patch-1

Refactor workflow input descriptions and cleanup steps

RudraPratik30 committed:
[docs]: Prometheus Plugin 3.16 (#14609)

RudraPratik30 committed:
[docs]: Branching and PR (#14601)

* [docs]: Branching and PR

* feedback updates

adishM98 committed:
Add workflows section to Helm documentation with configuration details for scheduling and Redis setup (#14620)

* Add workflows section to Helm documentation with configuration details for scheduling and Redis setup

* link update

---------

Co-authored-by: rudrapratik30 <pratik104agrawal@gmail.com>

RudraPratik30 committed:
[docs]: OpenAPI for API References (#14531)

* [docs]: OpenAPI for API References

* wip

* scim doc

* update openapi spec

* feedback update

adishM98 committed:
Update WEBSITE_SIGNUP_URL in cloud-frontend.yml

adishM98 committed:
Merge pull request #14629 from ToolJet/adishM98-patch-1

Update WEBSITE_SIGNUP_URL in cloud-frontend.yml

RudraPratik30 committed:
[docs]: Domain Change

RudraPratik30 committed:
config file changes

adishM98 committed:
Update WEBSITE_SIGNUP_URL in cloud-frontend.yml

adishM98 committed:
Merge pull request #14633 from ToolJet/adishM98-patch-1

Update WEBSITE_SIGNUP_URL in cloud-frontend.yml

RudraPratik30 committed:
[docs]: BigQuery Scope Update

adishM98 committed:
Merge pull request #14635 from ToolJet/docs/bigquery-scope

[docs]: BigQuery Scope Update

adishM98 committed:
Merge pull request #14631 from ToolJet/docs/domain-0change

[docs]: Domain Change

adishM98 committed:
Rename "Platform Ultimate Dashboard" to "Platform Metrics Dashboard" in observability documentation and update sidebar to include observability-otel setup.

adishM98 committed:
Add OpenTelemetry observability documentation and update sidebar reference

adishM98 committed:
Merge pull request #14638 from ToolJet/docs/correction-Observability

Rename "Platform Ultimate Dashboard" to "Platform Metrics Dashboard" in observability documentation and update sidebar to include observability-otel setup.

RudraPratik30 committed:
Merge branch 'develop' into docs/workflow-revamp

adishM98 committed:
Merge pull request #14216 from ToolJet/docs/workflow-revamp

[docs]: Workflow Revamp

RudraPratik30 committed:
Merge branch 'develop' into docs/smtp

RudraPratik30 committed:
update 3.5 and 3.16

adishM98 committed:
Merge pull request #12378 from ToolJet/docs/smtp

[docs]: Fix SMTP Attachment

slancerk committed:
Version 2 of Test-system

RudraPratik30 committed:
[docs]: Using PAT

slancerk committed:
removed view and made simple

slancerk committed:
small fix

slancerk committed:
added debug

slancerk committed:
eof issue fixed

slancerk committed:
updated manual docker build file

RudraPratik30 committed:
minor updates

prajwal-pai77 committed:
Merge develop

RudraPratik30 committed:
[docs]: Update API Server Gateway Domain (#14691)

slancerk committed:
updated docker pat in LTS and pre-release

RudraPratik30 committed:
[docs]: SAML Enable Group Sync UI

slancerk committed:
compiled in 2 jobs

adishM98 committed:
Merge pull request #14698 from ToolJet/docs/saml-group-sync

[docs]: SAML Enable Group Sync UI

RudraPratik30 committed:
[docs]: Authorize.net Plugin

adishM98 committed:
Merge pull request #14728 from ToolJet/docs/authorize.net

[docs]: Authorize.net Plugin

adishM98 committed:
Update fmt.Println message from 'Hello' to 'Goodbye'

adishM98 committed:
Merge pull request #14737 from ToolJet/adishM98-patch-1

Update cypress platform action

adityajoshi-08 committed:
[docs]: Added documentation for configuring SSO using Auth0 OIDC

adityajoshi-08 committed:
[docs]: Updated image borders for Auth0 OIDC documentation

adityajoshi-08 committed:
[docs]: Removed tip from import-libraries/python informing about running query on page load

adishM98 committed:
Merge pull request #14750 from ToolJet/docs/remove-python-install-tip

[docs]: Removed tip from import-libraries/python informing about run query on page load

RudraPratik30 committed:
[docs]: TJ API Updated End Points

RudraPratik30 committed:
move gitsync api

adishM98 committed:
[docs]: TJ API Updated End Points (#14758)

* [docs]: TJ API Updated End Points

* move gitsync api

RudraPratik30 committed:
[docs]: Small Fix in TJ API

adishM98 committed:
Merge pull request #14760 from ToolJet/docs/small-fix-tj-api

[docs]: Small Fix in TJ API

adityajoshi-08 committed:
[docs]: Updated images to follow highlights from the design convention and updated formatting from the style doc

RudraPratik30 committed:
[docs]: Dropdown Schema Example

adishM98 committed:
Merge pull request #14762 from ToolJet/docs/dropdown-v2-example-schema

[docs]: Dropdown Schema Example

adityajoshi-08 committed:
[docs]: Updated the development lifecycle docs and images to comply with the latest draft version

adityajoshi-08 committed:
[docs]: Added documentation for configuring stronger password validation rules

adityajoshi-08 committed:
[docs]: Updated documentation for configuring stronger password validation rules

adityajoshi-08 committed:
[docs]: Updated documentation for configuring stronger password validation rules - Added a few infos

adishM98 committed:
Added documentation for configuring stronger password validation rules. (#14779)

* [docs]: Added documentation for configuring stronger password validation rules

* [docs]: Updated documentation for configuring stronger password validation rules

* [docs]: Updated documentation for configuring stronger password validation rules - Added a few infos

RudraPratik30 committed:
[docs]: Update Readme for Domain Change

adishM98 committed:
Merge pull request #14783 from ToolJet/docs/update-readme-domain

[docs]: Update Readme for Domain Change

adishM98 committed:
Add disk space cleanup step in deployment workflow

Added a step to free up disk space before checking out the repository.

adishM98 committed:
Merge pull request #14787 from ToolJet/adishM98-patch-1

Add disk space cleanup step in deployment workflow

RudraPratik30 committed:
[docs]: NVM and NPM Version Update

adishM98 committed:
Merge pull request #14798 from ToolJet/docs/update-nvm-npm-version

[docs]: NVM and NPM Version Update

adityajoshi-08 committed:
[docs]: Added documentation for OIDC using non email identifier

adityajoshi-08 committed:
[docs]: Updated the environment variables for non email identifier in OIDC

nayanan10 committed:
[docs]: Added sendgrid image

nayanan10 committed:
[docs]: Updated filename in sendgrid document

adityajoshi-08 committed:
[docs]: Updated blurred screenshots and added a few pointers in draft version docs

adityajoshi-08 committed:
[docs]: Updated doc with lower resolution images

nayanan10 committed:
[docs] : Updated images size and texts bullentins

adityajoshi-08 committed:
[docs]: Updated explanation for OIDC login using Non email identifier.

RudraPratik30 committed:
[docs] : delete zone identifier file

adishM98 committed:
Merge pull request #14814 from ToolJet/docs/sendgrid

[docs]: Update SenGrid Plugin

adityajoshi-08 committed:
[docs]: Updated images with higher resolution ones

adityajoshi-08 committed:
[docs]: Updated images resolution in draft version docs

adityajoshi-08 committed:
[docs]: Updated images resolution in draft version docs

adityajoshi-08 committed:
[docs]: Updated gitsync docs - Added information about initial commit and Homepage URL

adityajoshi-08 committed:
[docs]: Styling changes in GitSync docs

adityajoshi-08 committed:
Merge branch 'develop' into docs/unique-oidc-id

adishM98 committed:
Merge pull request #14804 from ToolJet/docs/unique-oidc-id

[docs]: Added documentation for OIDC using non email identifier

adishM98 committed:
Merge pull request #14822 from ToolJet/docs/gitsync-initial-commit

[docs]: Updated gitsync docs - Added information about initial commit and Homepage URL

adityajoshi-08 committed:
[docs]: Added padding to smaller images

adityajoshi-08 committed:
[docs]: Add documentation for multiple OIDC Providers

adityajoshi-08 committed:
[docs]: Add docs for OIDC using Microsoft Entra ID

adityajoshi-08 committed:
[docs]: Add documentation for group sync using Microsoft Entra ID usign OIDC

adishM98 committed:
Merge pull request #14769 from ToolJet/docs/draft-version

[docs]: Updated the development lifecycle docs and images to comply with the latest draft version

adityajoshi-08 committed:
[docs]: Init OneLogin OIDC Setup

adityajoshi-08 committed:
[docs]: Add documentation for configuring OIDC using OneLogin

adityajoshi-08 committed:
[docs]: Formating changes to OneLogin OIDC docs

adishM98 committed:
Merge pull request #14642 from ToolJet/test-system-v2

Version 2 of Test-system

adityajoshi-08 committed:
[docs]: Microsoft entra id saml init

adityajoshi-08 committed:
[docs]: Added group sync docs to the OneLogin OIDC Configuration

adityajoshi-08 committed:
[docs]: Add docs for configuring SAML usign Microsoft Entra ID

adityajoshi-08 committed:
[docs]: Updated multi tenant docs with higher resolution images

adityajoshi-08 committed:
[docs]: Removed unnecessary padding from images

adishM98 committed:
Merge pull request #14828 from ToolJet/docs/multi-tenant-oidc

[docs]: Add documentation for multiple OIDC Providers

adishM98 committed:
docs: LOCKBOX_MASTER_KEY rotation script (#14856)

* docs: LOCKBOX_MASTER_KEY rotation script

* Clarify LOCKBOX_MASTER_KEY rotation instructions and improve formatting

* Update LOCKBOX_MASTER_KEY rotation instructions for serverless deployments

* Add collapsible sections for deployment instructions in LOCKBOX_MASTER_KEY rotation guide

* Removes key rotation timing and scenarios guidance

* Update prerequisites for LOCKBOX_MASTER_KEY rotation to specify database user access requirements

* Correct title casing for LOCKBOX_MASTER_KEY rotation guide and section headers

adishM98 committed:
Enhance Cypress workflow with permissions and debug steps

Updated Cypress workflow to include permissions and debugging steps. Removed deprecated environment variables and adjusted Cypress test configurations.

adishM98 committed:
Merge pull request #14868 from ToolJet/adishM98-patch-1

Enhance Cypress workflow with permissions and debug steps

RudraPratik30 committed:
[docs]: Trust Center (#14895)

* [docs]: Security Center

* de list docs v2

* materials to certifications

adityajoshi-08 committed:
[docs]: Added changes in Auth0 OIDC docs

adityajoshi-08 committed:
[docs]: Added changes to OneLogin OIDC docs

adityajoshi-08 committed:
[docs]: Added changes to Entra ID OIDC Docs

adityajoshi-08 committed:
[docs]: Updated example URL for Auth0 OIDC

adityajoshi-08 committed:
[docs]: Updated blurred image in Microsoft Entra ID OIDC doc

RudraPratik30 committed:
new structure

RudraPratik30 committed:
public application

RudraPratik30 committed:
pvt app wip

slancerk committed:
added to develop

adishM98 committed:
Merge pull request #14916 from ToolJet/render-docker-develop

added to develop

slancerk committed:
updated with LTS

slancerk committed:
removed recursive

adishM98 committed:
Merge pull request #14924 from ToolJet/render-docker-develop-v2.3

updated with LTS

slancerk committed:
v2.4 added free up space

adishM98 committed:
Merge pull request #14926 from ToolJet/render-docker-develop-v2.4

v2.4 added free up space

slancerk committed:
using pull_request

slancerk committed:
fixed render image issue

adishM98 committed:
Merge pull request #14930 from ToolJet/render-preview-develop

RudraPratik30 committed:
add code snippets

RudraPratik30 committed:
almost final

RudraPratik30 committed:
improved auth flow

adityajoshi-08 committed:
[docs]: Changes in auth0 docs as per ddoc review call

adityajoshi-08 committed:
[docs]: Changes in auth0 docs as per doc review call

adityajoshi-08 committed:
[docs]: Changes in auth0 docs as per doc review call

adityajoshi-08 committed:
[docs]: Changes in auth0 docs as per doc review call

RudraPratik30 committed:
final

RudraPratik30 committed:
[docs]: OpenAPI data source in the AI

adityajoshi-08 committed:
Merge branch 'sso-oidc' into docs/sso-docs-merged
Merge Auth0

adishM98 committed:
Merge pull request #14944 from ToolJet/docs/open-api-ai

[docs]: OpenAPI data source in the AI

adityajoshi-08 committed:
Merge branch 'docs/microsoft-entra-saml' into docs/sso-docs-merged
Merge Entra ID SAML

RudraPratik30 committed:
add link

adityajoshi-08 committed:
[docs] Merge OneLogin OIDC

adityajoshi-08 committed:
[docs]: Merged SSO docs in one branch

adishM98 committed:
Merge pull request #14647 from ToolJet/docs/pat-integration

[docs]: Embed ToolJet Application

adityajoshi-08 committed:
[docs]: Updated img classes and sidebar order

adishM98 committed:
Merge pull request #14945 from ToolJet/docs/sso-docs-merged

[docs]: Merge SSO docs into one PR

adishM98 committed:
Add Docker Compose and environment configuration for built-in and external PostgreSQL setups

RudraPratik30 committed:
[docs]: Editable Tags (#14894)

* [docs]: Editable Tags

* new updates

* formating changes

* auto pick chip color

adishM98 committed:
Change default branch in CI workflow to lts-3.16

adishM98 committed:
Merge pull request #14960 from ToolJet/adishM98-patch-1

Change default branch in CI workflow to lts-3.16

slancerk committed:
refined a bit

adityajoshi-08 committed:
[docs]: Add Observability Docs (#14871)

* [docs]: Add documentation for setting up observability using Datadog

* Updated doc flow for observability using datadog

* Updated doc flow for observability using datadog

* [docs]: Add docs for setting up observability using New Relic

* [docs]: Add docs for observability setup using Grafana

* Add intro line for setting up observability using Grafana docs

* [docs]: Add changes to Observability docs

* [docs]: Add documentation for observability using Last9

* [docs]: Fixed typo on setup page

* [docs]: Updated setup from observability docs

* [docs]: Changes in Observability docs wrt comments

* [docs]: Added screenshot for observability

adishM98 committed:
refined a bit (#14963)

RudraPratik30 committed:
[docs]: Update no. of data sources

adishM98 committed:
Merge pull request #14973 from ToolJet/docs/update-no-ds

[docs]: Update no. of data sources

adishM98 committed:
Add Vercel deployment workflow

This workflow automates the deployment of the ToolJet frontend to Vercel, including user authorization, Git checkout with submodules, dependency installation, and build steps.

adishM98 committed:
Merge pull request #14974 from ToolJet/adishM98-patch-1

Add Vercel deployment workflow

adishM98 committed:
Enhance ToolJet deployment documentation for version 3.16.0-LTS (#14896)

* Enhance ToolJet deployment documentation for version 3.16.0-LTS

- Updated Google Cloud Run setup to clarify architecture and environment variables.
- Improved Kubernetes setup documentation for AKS, EKS, GKE, and general Kubernetes, emphasizing the need for two separate PostgreSQL databases.
- Revised OpenShift deployment instructions to include detailed environment variable configurations and deployment steps.
- Added system requirements for PostgreSQL and Redis, specifying recommended versions and minimum specifications.
- Included warnings about database naming conflicts and critical configurations for successful deployment.

* Enhance upgrade documentation for ToolJet LTS version

- Added critical backup reminder for PostgreSQL instance before upgrading.
- Clarified database requirements, emphasizing the need for two separate database names (PG_DB and TOOLJET_DB).
- Included deployment flexibility options for database hosting.
- Updated installation notes to specify that the upgrade guide is only for existing installations.
- Improved formatting and clarity in various setup guides (Azure, DigitalOcean, Docker, ECS, Google Cloud Run, Kubernetes, OpenShift).

* Enhance deployment documentation for ToolJet on various platforms, including AWS AMI, Azure Container Apps, AWS ECS, Google Cloud Run, Kubernetes (AKS, EKS, GKE), and Openshift. Updated warnings and notes regarding PostgreSQL database setup and ToolJet AI features.

* Enhance deployment documentation for ToolJet LTS version 3.16.0 by updating backup instructions, removing redundant version requirements, and clarifying Redis configuration for external instances.

* Refine deployment documentation for ToolJet by removing redundant environment variable references and adding notes on securing Redis passwords across various platforms including AWS AMI, Docker, Google Cloud Run, Kubernetes (AKS, EKS, GKE), and OpenShift.

* Refine Redis configuration instructions in AWS AMI deployment documentation for clarity and formatting consistency.

* Add new environment variables for bulk upload limits in deployment documentation

* Refine deployment documentation for ToolJet subpath installation by improving environment variable formatting and clarifying upgrade prerequisites.

* Update PostgreSQL database links in deployment documentation for consistency across ECS, Helm, Kubernetes (AKS, EKS, GKE), and OpenShift setups.

* Remove warning about whitelisting ToolJet AI features from Helm deployment documentation

* Refine environment variable configuration for workflow scheduling in Helm deployment documentation

* formatting updates till gke

* final formatting updates

---------

Co-authored-by: rudrapratik30 <pratik104agrawal@gmail.com>

RudraPratik30 committed:
[docs]: App Builder Sprint 21 (#14983)

* audio and camera docs

* iframe

* iframe and tabs component

slancerk committed:
Added pre-release support

adishM98 committed:
Added pre-release support (#15005)

slancerk committed:
ToolJet host changed to 80

slancerk committed:
added platform tag for tooljet and updated postgres 13 -> 16

RudraPratik30 committed:
[docs]: App Builder Sprint (#14879)

* [docs]: App Builder Sprint

* minor formatting

* html and reset query

* form styles

* minor updates

* Components UI/UX style enhancements

* table img updates

RudraPratik30 committed:
[docs]: Dynamic Access Rules

RudraPratik30 committed:
[docs]: Update Marketplace and Deployment Overview (#15016)

* [docs]: Update Marketplace Overview and Deployment Overview

* fix build issues

adishM98 committed:
Merge pull request #15030 from ToolJet/docs/app-builder-permissions

[docs]: Dynamic Access Rules

RudraPratik30 committed:
[docs]: Improve Navigation Sidebar (#15047)

* nav sidebar label and auto open

* paid and self hosted tags

* navbar icons

* correct typo

gsmithun4 committed:
Refactor code structure for improved readability and maintainability

adishM98 committed:
Merge pull request #14950 from ToolJet/fix/ce-dockerfile-compose

Add Docker Compose and environment configuration for built-in and external PostgreSQL setups

RudraPratik30 committed:
[docs]: Update NPM and NVM version (#15049)

RudraPratik30 committed:
[docs]: Delete unwanted docs (#15078)

gsmithun4 committed:
Merge pull request #15066 from ToolJet/rebase/develop-main

Merge main to develop

gsmithun4 committed:
Merge branch 'develop' into feature/add-couchbase-support
